source/developers/projects/studio/api/repo/push-to-remote.rst

```diff
@@ -0,0 +1,89 @@
+.. _crafter-studio-api-repo-push-to-remote:
+
+==============
+Push To Remote
+==============
+
+Push content to remote repository from site content repository.
+
+--------------------
+Resource Information
+--------------------
+
++----------------------------+-------------------------------------------------------------------+
+|| HTTP Verb                 || POST                                                             |
++----------------------------+-------------------------------------------------------------------+
+|| URL                       || ``/api/1/services/api/1/repo/push-to-remote.json``               |
++----------------------------+-------------------------------------------------------------------+
+|| Response Formats          || ``JSON``                                                         |
++----------------------------+-------------------------------------------------------------------+
+|| Required Role             || Admin, site admin                                                |
++----------------------------+-------------------------------------------------------------------+
+
+----------
+Parameters
+----------
+
++----------------------+-------------+----------------+--------------------------------------------------------------+
+|| Name                || Type       || Required      || Description                                                 |
++======================+=============+================+==============================================================+
+|| site_id             || String     || |checkmark|   || Site to use                                                 |
++----------------------+-------------+----------------+--------------------------------------------------------------+
+|| remote_name         || String     || |checkmark|   || Remote repository name                                      |
++----------------------+-------------+----------------+--------------------------------------------------------------+
+|| remote_branch       || String     || |checkmark|   || Branch from which content will be pulled                    |
```

Comment:
Is `remote_branch` here supposed to be `Branch from which content will be **pushed**`?